CHANGELOG.md

```diff
@@ -15,14 +15,10 @@
 
 * **[Feature]** Now supports tvOS.
 
-___
-
-## Version 2.2.0 (Under development)
-
 ### AppCenterData
 
 * **[Feature]** Add support for offline list of documents.
-
+* **[Feature]** Change the default offline persistence from 1 day to infinite.
```

Comment:
Let's use the same terminology `time-to-live (TTL)` as in the docs (https://docs.microsoft.com/en-us/appcenter/sdk/data/ios).

```suggestion
* **[Feature]** Change the default device time-to-live (TTL) from 1 day to infinite (never expire).
```

Reply:
woo.. how about remove `device` in this sentence? :)